rtgui/editorpanel.cc

```diff
@@ -46,6 +46,8 @@ class EditorPanel::MonitorProfileSelector
     void prepareProfileBox ()
     {
         profileBox.set_size_request (100, -1);
+        profileBox.setPreferredWidth(90, 150);
```

Comment:
Well I agree that is probably not important (actually I strongly dislike it as I am not used to it ;-)), but it is definitely all over the code and I did try to be consistent with that up to now. While that is probably irrelevant to this pull request, I think we should fix one way and make `astyle` apply that for us instead of leaving it untouched.